packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -99,7 +99,7 @@ function ChangelogTooltip({
       <InteractivePopup
         {...props}
         position={InteractivePopupPosition.MainFeedLeftEnd}
-        className="ml-6 border shadow-2 focus:outline-none w-[24rem] max-w-[360px] max-h-[466px] bg-theme-bg-tertiary border-theme-color-cabbage"
+        className="ml-6 border shadow-2 focus:outline-none w-[24rem] max-w-[360px] bg-theme-bg-tertiary border-theme-color-cabbage"
```

Comment:
I just noticed we are using `px` value for the max width. Should we fix this? cc @dailydotdev/web-team

Reply:
All cool seems to be a legacy stuff as it was not introduced here 🤘